portfolio/src/main/webapp/comments-script.js

```diff
@@ -51,3 +54,8 @@ function createRowElement() {
     const rowElement = document.createElement('tr');
     return rowElement;
 }
+
+function changeCommentsPerPage(){
```

Comment:
Why is "changeCommentsPerPage" decrementing by one?

Reply:
Ahh since it's zero indexed. I'd renamed this or add a comment to make it more clear.